### PR TITLE
Bump MSRV version in CI

### PR DESCRIPTION
**Pull request by stevenroose**

### CODE REVIEW

**Comment by stevenroose** [comment]
I noticed we have both appveyor and Travis here and that doesn't seem necessary.

File: appveyor.yml [hunk 1]
```diff
@@ -45,4 +45,4 @@ build: false
 
 test_script:
   - cargo generate-lockfile --verbose && cargo update -p cc --precise "1.0.41" --verbose
-    cargo test
+  - cargo test
```
Comment (dr-orlovsky) [inline appveyor.yml:48]: This will conflict withchanges introduced by #27

**Review by dr-orlovsky — CHANGES_REQUESTED** [review]
Conflicts with #27 - see comment

**Comment by stevenroose** [comment]
Updated

**Review by dr-orlovsky — APPROVED** [review]

**Review by TheBlueMatt — APPROVED** [review]

**Pull request merged**